src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/behavior/ov_plugin/properties.cpp

```diff
@@ -62,28 +43,10 @@ const std::vector<ov::AnyMap> auto_batch_properties = {
 
 INSTANTIATE_TEST_SUITE_P(ov_plugin, OVPropertiesTests,
         ::testing::Combine(
-                ::testing::Values(ov::test::conformance::targetDevice),
+                ::testing::ValuesIn(return_all_possible_device_combination()),
                 ::testing::ValuesIn(default_properties)),
         OVPropertiesTests::getTestCaseName);
 
-INSTANTIATE_TEST_SUITE_P(ov_plugin_Hetero, OVPropertiesTests,
```

Comment:
Why did you reduce the number of tests?

Reply:
I didn't reduce. I just replaced instantiations by `return_all_possible_devices`